cpp/dolfinx/fem/CoordinateElement.cpp

```diff
@@ -5,30 +5,59 @@
 // SPDX-License-Identifier:    LGPL-3.0-or-later
 
 #include "CoordinateElement.h"
+#include <basix.h>
 #include <unsupported/Eigen/CXX11/Tensor>
 
 using namespace dolfinx;
 using namespace dolfinx::fem;
 
 //-----------------------------------------------------------------------------
 CoordinateElement::CoordinateElement(
-    mesh::CellType cell_type, int topological_dimension,
-    int geometric_dimension, const std::string& signature,
-    const ElementDofLayout& dof_layout, bool is_affine,
-    const std::function<int(double*, int, int, const double*)>&
-        evaluate_basis_derivatives)
-    : _tdim(topological_dimension), _gdim(geometric_dimension),
-      _cell(cell_type), _signature(signature), _dof_layout(dof_layout),
-      _is_affine(is_affine),
-      _evaluate_basis_derivatives(evaluate_basis_derivatives)
+    int basix_element_handle, int geometric_dimension,
```

Comment:
What is `basix_element_handle`?

Reply:
It's the identifier for a basix finite element.

Reply:
As far I can understand Basix only creates the finite element once, as it has to invert a matrix for each element, thus this is cached and identified through an integer.

Reply:
I've requested documentation of this in FEniCS/basix#98